packages/frontend/component/src/ui/slider/slider.tsx

```diff
@@ -0,0 +1,52 @@
+import * as Sliders from '@radix-ui/react-slider';
+import { useRef } from 'react';
+
+import * as styles from './index.css';
+
+export interface SliderProps extends Sliders.SliderProps {
+  nodes?: number[]; // The values where the nodes should be placed
+}
+
+export const Slider = ({
```

Comment:
It's a bit difficult to click/drag because the response area is too small. Maybe we can implement this kind of layout to increase the response area. And provide variables for easy adjustment

![CleanShot 2024-08-19 at 08.54.51@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/LakojjjzZNf6ogjOVwKE/c48086b1-cd5f-4bea-89a1-664969c1954f.png)